Funcky.Async/Extensions/AsyncEnumerableExtensions/Shuffle.cs

```diff
@@ -0,0 +1,17 @@
+using static Funcky.Internal.Mixer;
+
+namespace Funcky.Async.Extensions
+{
+    public static partial class AsyncEnumerableExtensions
+    {
+        /// <summary>
+        /// Returns the given sequence in random Order in O(n).
+        /// </summary>
+        /// <typeparam name="TSource">The type of the elements in the enumerable.</typeparam>
+        /// <remarks>This method is implemented by using deferred execution. The immediate return value is an object that stores all the information that is required to perform the action. The query represented by this method is not executed until the object is enumerated either by calling its GetEnumerator method directly or by using foreach.</remarks>
+        [Pure]
+        public static async ValueTask<IEnumerable<TSource>> Shuffle<TSource>(this IAsyncEnumerable<TSource> source)
```

Comment:
We should accept a cancellation token

Reply:
Fixed in fa3d555bd5dbcfabc158ec84cf882d7eef38f363